addons/binding/org.openhab.binding.yeelight/ESH-INF/config/config.xml

```diff
@@ -1,21 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <config-description:config-descriptions
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
-    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
         http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
-    <config-description uri="thing-type:device:base">
-        <parameter name="deviceId" type="text" required="true">
-		    <label>Device ID</label>
-		    <description>Id of the Yeelight device to connect with.</description>
+	<config-description uri="thing-type:device:base">
+		<parameter name="deviceId" type="text" required="true">
+			<label>Device ID</label>
+			<description>Id of the Yeelight device to connect with.</description>
 		</parameter>
-		        <parameter name="duration" type="integer" required="true" min="50" max="5000">
-            <label>Duration</label>
-            <description>Duration of transition of events such as on/off, change of brightness and change of color , in milliseconds.</description>
-            <unitLabel>milliseconds</unitLabel>
-            <default>500</default>
-        </parameter>
-    </config-description>
-
+		<parameter name="duration" type="integer" required="true" min="50" max="10000">
```

Comment:
Shouldn't min be `0` for immediate effect?

Reply:
Min is 50, according to Yeelight specs. 0 Will work, but anything below 50 is treated as 50.

Reply:
Alright